prediction_market_agent/db/db_storage.py

```diff
@@ -0,0 +1,61 @@
+import json
+from typing import Any, Dict, Sequence
+
+from loguru import logger
+from prediction_market_agent_tooling.tools.utils import utcnow
+from sqlmodel import Session, SQLModel, create_engine, desc, select
+
+from prediction_market_agent.db.models import LongTermMemories
+from prediction_market_agent.utils import DBKeys
+
+
+class DBStorage:
+    def __init__(self, sqlalchemy_db_url: str | None = None):
+        if not sqlalchemy_db_url:
+            keys = DBKeys()
+            if not keys.sqlalchemy_db_url:
+                raise EnvironmentError(
+                    "Cannot initialize DBHandler without a valid sqlalchemy_db_url"
+                )
+            sqlalchemy_db_url = keys.sqlalchemy_db_url
+
+        self.engine = create_engine(sqlalchemy_db_url)
+
+    def _initialize_db(self) -> None:
+        """
+        Creates the tables if they don't exist
+        """
+
+        # trick for making models import mandatory - models must be imported for metadata.create_all to work
```

Comment:
Is this necessary? You are doing `from prediction_market_agent.db.models import LongTermMemories` above.

Reply:
If I remove this, isort or mypy removes the import and it creates no tables.
Other option is to do `import *`, which mypy does not like.
Hence cheating here and logging this.